mw-webapp/src/component/table/Table.module.scss

```diff
@@ -6,29 +6,54 @@ $cell-max-width: 600px;
 .table {
   width: 100%;
   height: 100%;
-  border: $table-border;
+  border-radius: $primaryBorderRadius;
   background-color: var(--primaryBgColor);
-  border-collapse: collapse;
-}
+  border-collapse: separate;
+  border-spacing: 0;
 
-.table th {
-  padding: 5px;
-  border: $table-border;
-  background: var(--primaryBgColor);
-  background-clip: padding-box;
-  font-weight: bold;
-}
+  th {
+    padding: 5px;
+    border-top: $table-border;
+    border-right: $table-border;
+    border-bottom: $table-border;
+    background: var(--primaryBgColor);
+    background-clip: padding-box;
+    font-weight: bold;
+  }
+
+  td {
+    max-width: $cell-max-width;
+    padding: 8px 10px;
+    border-right: $table-border;
+    border-bottom: $table-border;
+    text-align: left;
+    vertical-align: top;
+    word-wrap: break-word;
 
```

Comment:
Can you please explain how these changes connect with issue - add border-radius?

Reply:
It is not directly connected with "add border-radius". It just refactors code to use SCSS nested structures for reducing number of repeated selecters. It's done in a separate commit. I can revert commit if you think it's not suitable for this PR.

Reply:
It's okey.
But what do you think about using
border: $table-border $table-border $table-border  0;

instead of:
border-top: $table-border;
border-right: $table-border;
border-bottom: $table-border;

Reply:
It's impossible to do like this because border accepts [args](https://developer.mozilla.org/en-US/docs/Web/CSS/border#formal_syntax): line-width, line-style, color.

But I found another way to make it shorter:
```css
border: $table-border;
border-left: 0;
```
Are you ok with that?

Reply:
Done